# scRNA-tutorial

### 单细胞学习资料

1. [剑桥大学课程Analysis of single cell RNA-seq data](https://www.singlecellcourse.org/index.html)
2. [哈佛大学教程，很详细](https://hbctraining.github.io/scRNA-seq/)
3. https://github.com/quadbio/scRNAseq_analysis_vignette/blob/master/Tutorial.md R语言教程，很不错
4. [Single-cell best practices](https://www.sc-best-practices.org/preamble.html)
5. https://swaruplab.bio.uci.edu/tutorial/tutorial.html 暂时看着还不错
6. [scRNAseq-analysis-notes](https://github.com/crazyhottommy/scRNAseq-analysis-notes)
7. [awesome-single-cell](https://github.com/seandavi/awesome-single-cell)
8. [GPU-Accelerated Single-Cell RNA Analysis with RAPIDS-singlecell](https://developer.nvidia.com/blog/gpu-accelerated-single-cell-rna-analysis-with-rapids-singlecell/) GPU加速
9. [cellranger官方资料](https://www.10xgenomics.com/cn/support/software/cell-ranger/latest/tutorials/cr-tutorial-ct)
10. [ANALYSIS OF SINGLE CELL RNA-SEQ DATA](https://broadinstitute.github.io/2020_scWorkshop/)  Broad Institute
11. https://biocellgen-public.svi.edu.au/mig_2019_scrnaseq-workshop/trajectory-inference.html
12. [Orchestrating Single-Cell Analysis with Bioconductor](https://bioconductor.org/books/3.12/OSCA/)
13. [Advanced Single-Cell Analysis with Bioconductor](https://bioconductor.org/books/3.14/OSCA.advanced/index.html)
14. [Basics of Single-Cell Analysis with Bioconductor](https://bioconductor.org/books/3.13/OSCA.basic/quality-control.html#common-choices-of-qc-metrics)
15. https://hds-sandbox.github.io/AdvancedSingleCell/   四个莲花样本（两个感染根瘤菌和两个野生型）
16. https://www.earlham.ac.uk/
17. [awesome-deep-learning-single-cell-papers](https://github.com/OmicsML/awesome-deep-learning-single-cell-papers)
18. [In-depth-NGS-Data-Analysis-Course](https://github.com/hbctraining/In-depth-NGS-Data-Analysis-Course/blob/master/sessionIV/schedule/README.md)
19. https://nbisweden.github.io/single-cell_sib_scilifelab_2021/schedule.html
20. [刘小乐生信教程](https://liulab-dfci.github.io/bioinfo-combio/)
21. https://rockefelleruniversity.github.io/scRNA-seq/
22. [剑桥大学，不同条件下比较](https://bioinformatics-core-shared-training.github.io/UnivCambridge_ScRnaSeq_Nov2021/)
23. [北大Yuting Liu](https://bookdown.org/ytliu13207/SingleCellMultiOmicsDataAnalysis/)

### Paper
1. [Cell-type-specific responses to fungal infection in plants revealed by single-cell transcriptomics](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(23)00344-X?returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS193131282300344X%3Fshowall%3Dtrue#) 植物-真菌感染，有代码
3. [Accelerating single-cell genomic analysis with GPUs](https://www.biorxiv.org/content/10.1101/2022.05.26.493607v1.full) GPU加速单细胞数据分析
4. [使用高通量单细胞 RNA 测序揭示拟南芥根的时空发育轨迹](https://www.cell.com/developmental-cell/fulltext/S1534-5807(19)30145-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1534580719301455%3Fshowall%3Dtrue#secsectitle0085)
5. [A comparison of single-cell trajectory inference methods](https://www.nature.com/articles/s41587-019-0071-9#code-availability)
6. [使用空间和单细胞技术了解植物病原体相互作用](https://www.nature.com/articles/s42003-023-05156-8) 很不错为文献
7. [单细胞拟南芥根图谱揭示了野生型和细胞身份突变体的发育轨迹](https://www.cell.com/developmental-cell/fulltext/S1534-5807(22)00033-8?returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1534580722000338%3Fshowall%3Dtrue#secsectitle0010) 有代码
8. [根韧皮部极细胞图谱揭示了原韧皮部相邻细胞中常见的转录状态](https://www.nature.com/articles/s41477-022-01178-y#code-availability) 有代码
9. [Time-resolved single-cell and spatial gene regulatory atlas of plants under pathogen attack](https://www.biorxiv.org/content/10.1101/2023.04.10.536170v2.full) 有代码
10. [拟南芥叶对丁香假单胞菌感染的单细胞分析](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)00687-3?returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124723006873%3Fshowall%3Dtrue#sectitle0035) 有代码
11. [跨模式单细胞分析的最佳实践](https://www.nature.com/articles/s41576-023-00586-w)
12. [Understanding plant pathogen interactions using spatial and single-cell technologies](https://www.nature.com/articles/s42003-023-05156-8) NC


### 数据库
- [Arabidopsis thaliana: root](https://bioit3.irc.ugent.be/plant-sc-atlas/root/download)  拟南芥根部


### QC


### Doublet
- [DoubletFinder](https://github.com/chris-mcginnis-ucsf/DoubletFinder)

### Annotation
- [singleR](https://bioconductor.org/books/release/SingleRBook/sc-mode.html)
- [Garnett](https://cole-trapnell-lab.github.io/garnett/)
- [Capybara: A computational tool to measure cell identity and fate transitions](https://github.com/morris-lab/Capybara)
- [Tutorial: guidelines for annotating single-cell transcriptomic maps using automated and manual methods](https://www.nature.com/articles/s41596-021-00534-0) Nature protocols


### Cluster
- [使用 Seurat 进行 countsplit 来找到最佳簇数](https://github.com/anna-neufeld/countsplit/issues/8) Tommy推荐
- [MultiK：一种用于确定单细胞 RNA 测序数据中最佳簇数的自动化工具](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02445-5)
- [Cytocipher - detection of significantly different cell populations in scRNA-seq](https://github.com/BradBalderson/Cytocipher)
- [scRNAseq clustering significance test](https://divingintogeneticsandgenomics.com/post/scrnaseq-clustering-significant-test-an-unsolvable-problem/)
- [icking optimal resolution for single cell in seurat pipeline](https://www.biostars.org/p/9576814/#9576818)
- [Finding optimal cluster resolution in Seurat](https://github.com/satijalab/seurat/issues/1565)  seurat官方issues
- [使用基于子采样的稳健性指标选择单细胞聚类参数值](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03957-4) paper
- [In-depth-NGS-Data-Analysis-Course](https://hbctraining.github.io/In-depth-NGS-Data-Analysis-Course/sessionIV/lessons/SC_clustering_analysis.html)

### Pseudotime
- [Monocle](https://cole-trapnell-lab.github.io/monocle-release/)
- [Trajectory inference](https://biocellgen-public.svi.edu.au/mig_2019_scrnaseq-workshop/trajectory-inference.html)


